tex/set-theory/models.tex

```diff
@@ -693,9 +716,15 @@ \section{Picturing inner models}
 \end{dproblem}
 
 \begin{sproblem}
-	[Inaccessible cardinals produce models]
+	[Strongly inaccessible cardinals produce models]
 	\label{prob:inaccessible_model}
 	\gim
-	Let $\kappa$ be an inaccessible cardinal.
+	Let $\kappa$ be a strongly inaccessible cardinal.
 	Prove that $V_\kappa$ is a model of $\ZFC$.
+	\begin{hint}
+		Use \Cref{lem:transitive_ZFC}.
+		To prove $V_\kappa \vDash \PowerSet$ you need $\kappa$ to be a strong limit cardinal,
+		and to prove $V_\kappa \vDash \Replacement$ you need $\kappa$ to be inaccessible ---
+		this is why we cared about cofinality and inaccessibility.
```

Comment:
(1) Try to motivate the reason why we cared about "cofinality" and "M-definable with parameter m₁, …, mₙ in M", although possibly placed in awkward places.